packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -717,6 +717,76 @@ class PhysicalModel extends SingleChildRenderObjectWidget {
   }
 }
 
+class PhysicalShape extends SingleChildRenderObjectWidget {
+  /// Creates a physical model with a rounded-rectangular clip.
+  ///
+  /// The [color] is required; physical things have a color.
```

Comment:
"physical things have a color" seems trite and it begs some questions:
- Why isn't this more like BoxDecoration with background color, gradient, image...
- How does one configure the color of the border itself? Or more generally how does one configure the border's paint?

Reply:
I guess there is no support for anything but background color as there was no need for it so far (note that this is almost a 1:1 copy of PhysicalModel).

As for color for the border itself - we don't currently draw a border not in PhysicalModel and not here... I guess we never needed to...? do we need it now?